client/src/attestation.rs

```diff
@@ -155,6 +181,93 @@ impl AttestationPayload {
 	}
 }
 
+/// Attestation record.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct AttestationRecord {
+	/// Ethereum address of the peer being rated.
+	about: String,
+	/// Unique identifier for the action being rated.
+	key: String,
+	/// Given rating for the action.
+	value: String,
+	/// Optional field for attaching additional information to the attestation.
+	message: String,
+	/// The 'r' value of the ECDSA signature.
+	sig_r: String,
+	/// The 's' value of the ECDSA signature.
+	sig_s: String,
+	/// Recovery id of the ECDSA signature.
+	rec_id: String,
+}
```

Comment:
Do you think storage.rs would be a better place for this, since we already keep ScoreRecord there?

Reply:
I agree!